mackerel.go

```diff
@@ -24,6 +24,10 @@ type Client struct {
 	UserAgent         string
 	AdditionalHeaders http.Header
 	HTTPClient        *http.Client
+
+	// ErrorLog specifies an optional logger.
```

Comment:
What does `ErrorLog` mean (maybe `Logger`)?

Reply:
I will fix this soon...